src/screens/LinkContextScreen/index.js

```diff
@@ -182,7 +182,7 @@ const styles = StyleSheet.create({
   },
   bottomAnchorSwipeText: {
     fontFamily: fonts.inter[500],
-    color: COLORS.white2,
+    color: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of `bottomAnchorSwipeText` has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change doesn't affect the readability and contrast ratio on different screens, especially if the background color is light. If `COLORS.white2` was a lighter shade than `COLORS.white`, it might have been used to provide better contrast against certain backgrounds.
```diff
-    color: COLORS.white2,
+    color: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->